exp-player/addon/styles/components/isp-radio-group.scss

```diff
@@ -24,6 +24,11 @@
   margin-top: -15px;
 }
 
-.label-spacing {
-  max-width: 6%;
+.measure-four {
+  width: 6%;
```

Comment:
If two sets of scales had the same number of options, why did the formatting "look off" for one? Is there an example?

Without specific examples to see, just as general notes: what I've found in the past is that if two things were otherwise identical, then if you had to "special case" the display of one of them (with extra styles), it meant there was some basic underlying difference responsible for the display difference. The more specific trial-and-error display tweaks you have to make, the more likely it is that your page display is locked to a set of assumptions that could break for someone else. (smaller monitor, larger text, etc)